package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mage-worlds-apart",
+  "version": "1.0.0",
+  "description": "Wiki for this Chronicle",
+  "main": "index.js",
+  "scripts": {
```

Comment:
You will need to add `gaming-wiki-client` as a [workspace](https://docs.npmjs.com/cli/v9/using-npm/workspaces).

Reply:
Did we do this?

Reply:
npm init -w ./packages/a

Is this the right command in the terminal? I don't understand what hte 'a' means. Does it apply to my work or should it be changed and to what?